definition-map.js

```diff
@@ -140,6 +140,12 @@ const MUTATIONS = {
             email: "The email of the Current User attempting to log in.",
             password: "The password of the Current User attempting to log in.",
         }
+    },
+    SelectCurrentUserActiveLicense: {
+        metadescription: "Changes the current users active license.",
+        args: {
+            licenseId: "The licenseId of the license that is to be assigned to the currentUser.",
```

Comment:
Not sure 'assigned' is the best word, as the user will already be assigned the license and isn't going to be given a new one. Maybe something like "The ID of the License that should be set to 'active' for the CurrentUser."?